### PR TITLE
publish 2.0.5 to play!

### PR DESCRIPTION
**Pull request by bleehu**

### CODE REVIEW

File: CharacterCreation/01_Races.txt [hunk 1]
```diff
@@ -203,9 +205,47 @@ Once at level 3, and every 5 levels thereafter a Cyborg may purchase an
...
+Ability: Bioluminesce - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
+
+Chondrich
+-1 to base CHA, +1 to both STR and FOR
+Tough Skinned: Natural Armor protects you in a firefight. +10 armor coverage, + 5+(FOR/2) Damage reduction. Armor piercing rounds ignore these bonuses. 
+Large: You are larger than your counterparts and are thus easier to hit, -5 DC on any chance to hit you.
+
+Mendar
++2 to base CHA
+-2 to chosen stats.
```
Comment (1sourcecontrol) [inline CharacterCreation/01_Races.txt:238]: -2 to 1 chosen stat? Or a -1 to 2 different stats?

File: Basic Rules.txt [hunk 2]
```diff
@@ -568,7 +568,24 @@ reduced by 1, permanently (repairable, DC 60% Repair check), not stackable.
...
+Buzzed (drunk) - Many drinks in the galaxy contain a substance called alchohol.
+	When consuming these drinks, the effect should list a FORT save to avoid
+	increasing your level of intoxication. The first level of intoxication is Buzzed.
+	While buzzed, you gain a +10% to miss with any attack and a temporary -1
+	to Perception. At the same time, you get a +1 to charisma and luck. You also
+	take a -20 to skill checks involving driving, flying or opperating heavy 
+	machinery. If you fail a Fort save to prevent increasing your intoxication
+	level while buzzed, you become hammered. As long as you don't become 
+	hammered, being buzzed will wear off in two hours.
+Hammered (drunk) - While hammered, you take a -30 to accuracy rolls when
```
Comment (1sourcecontrol) [inline Basic Rules.txt:580]: should say +30% miss chance, to be consistent

**Comment by 1sourcecontrol** [comment]
Looks pretty solid!

File: CharacterCreation/01_Races.txt (hunk 1 above)
Comment (bleehu) [reply CharacterCreation/01_Races.txt:238]: It does say stats plural. Is there a balance issue there that I'm not seeing?

Comment (1sourcecontrol) [reply CharacterCreation/01_Races.txt:238]: I see ambiguity with saying they can assign a number to stats, but not defining the quantity of stats that they assign to. For example, -2 to 5 chosen stats, -2 to infinite chosen stats, or -1 to 2 chosen stats with the ability to use the -1 twice on the same stat, etc.

File: Basic Rules.txt (hunk 2 above)
Comment (1sourcecontrol) [reply Basic Rules.txt:580]: @bleehu why didnt we fix this? It says -30 accuracy

**Pull request merged**